src/Transport/Tcp/SslContext.cs

```diff
@@ -6,7 +6,7 @@ class SslContext
     {
         private String sslProtocol;
 
-        public SslContext() : this("Tls")
+        public SslContext() : this("None")
```

Comment:
Since  this value should correspond the SslProtocols enum,
 I would suggest to replace the string with SslProtocols.None.ToString()